python/ray/data/_internal/stats.py

```diff
@@ -62,10 +62,13 @@ def __init__(self, stage_name: str, parent: "DatasetStats"):
     def build_multistage(self, stages: StatsDict) -> "DatasetStats":
         stage_infos = {}
         for i, (k, v) in enumerate(stages.items()):
-            if i == 0:
-                stage_infos[self.stage_name + "_" + k] = v
+            if len(stages) > 1:
```

Comment:
This change just allows build_multistage() to be used in all scenarios as a simplification.